autonomica-api/app/main.py

```
@@ -1271,6 +1271,1113 @@
             "error": str(e)
         }
 
+# --- SEO Research and Analysis Endpoints ---
+
+@app.post("/api/seo/analyze")
+async def analyze_keyword(
+    request: dict,
+    current_user: ClerkUser = Depends(get_current_user)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)